flag.go

```diff
@@ -881,7 +887,29 @@ func (f *FlagSet) parseArgs(args []string) (err error) {
 func (f *FlagSet) Parse(arguments []string) error {
 	f.parsed = true
 	f.args = make([]string, 0, len(arguments))
-	err := f.parseArgs(arguments)
+	err := f.parseArgs(arguments, nil)
```

Comment:
Make the user function take `(flag, value, origArg)` ? Or even don't pass `origArg` at all to the user functionl? When you 'rebuild' your command line using this function it would translate `-abcd` into `-a -b -c -d` and `-oarg` into `-o arg` but who cares?